pnpm-lock.yaml

```diff
@@ -189,6 +189,1615 @@ importers:
         specifier: ^1.2.2
         version: 1.6.0(@types/node@22.2.0)(jsdom@22.1.0)
 
+  packages/eslint-plugin-wdio:
```

Comment:
This file should not be in here, we accidentally added it in on of our last PRs. Can you update your branch and call `npm install` again? Make sure it applies changes to `package-lock.json`

Reply:
@christian-bromann Updated